### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by dlespiau** [TypeStrong/typedoc#1068]
- [x] I have checked [issues with enhancement label](https://github.com/TypeStrong/typedoc/labels/enhancement) and found no duplicates

## Problem

There are a few interesting commits in master (typescript 3.4.x, dependency updates with `npm audit --fix` that it'd be nice to have in an release. It has been noted in other issues too:

- https://github.com/TypeStrong/typedoc/issues/1009#issuecomment-483314308
- https://github.com/TypeStrong/typedoc/issues/1015#issuecomment-491971435

## Suggested Solution

Release a new version!

**Comment by dlespiau** [comment]
You may want to update `shelljs` before making a new release: https://github.com/TypeStrong/typedoc/issues/1056.

**Comment by GianlucaMaggio** [comment]
Please can you publish on NPM?

**Comment by fr0** [comment]
Is this project dead? The last release was published 5 months ago.

**Comment by bitjson** [comment]
For anyone interested here, I just published the latest master to `@bitjson/typedoc` (to get an update out for [typescript-starter](https://github.com/bitjson/typescript-starter)). 

You can install with `npm install @bitjson/typedoc`.

**Comment by Gerrit0** [comment]
The project isn't dead, but @aciccarello and I are the only two members still working on it and the occasional other PR submitted by a user. Anthony is the only one with publish permissions on npm, so a release is waiting on him. He hasn't had time for TypeDoc recently, and was having trouble with the build last time he tried to put a release out.

---

@bitjson your published version still depends on `typedoc-default-themes` which has received several updates since last published. If you want to stay on the latest changes for both themes and the main repo, you'll also need to track that repo.

I keep `@gerrit0/typedoc` up to date with master with the dependencies properly synced.

**Comment by fr0** [comment]
Thanks for your hard work, @Gerrit0. I might suggest that @aciccarello should give you publish permissions if he doesn't have time to devote to this project. A lot of people are relying on this project, and it deserves some time and attention.

**Comment by aciccarello** [comment]
[v0.15.0](https://github.com/TypeStrong/typedoc/releases/tag/v0.15.0) has been released! 🎉 

Thanks everyone for your patience.

**Comment by dlespiau** [comment]
Thank you so much!
